### PR TITLE
Convert asserts

### PR DESCRIPTION
**Pull request by Alex286756**

### CODE REVIEW

File: src/test/java/com/github/fppt/jedismock/TestCommandParser.java [hunk 1]
```diff
@@ -63,23 +68,23 @@ public void testConsumeCount2() {
...
-        assertEquals(cmd.parameters().get(2).toString(), "ef");
+        assertThat(cmd.parameters().get(0).toString()).isEqualTo("");
+        assertThat(cmd.parameters().get(1).toString()).isEqualTo("abcd");
+        assertThat(cmd.parameters().get(2).toString()).isEqualTo("ef");
     }
 
     @Test
     public void testConsumeCharacterError() throws ParseErrorException {
         InputStream stream = new ByteArrayInputStream("".getBytes());
         try {
```
Comment (inponomarev) [inline src/test/java/com/github/fppt/jedismock/TestCommandParser.java:85]: Here and in all the similar places it should be `assertThatThrownBy`

File: src/test/java/com/github/fppt/jedismock/comparisontests/lists/BlockingOperationsTest.java [hunk 2]
```diff
@@ -161,12 +158,12 @@ public void whenUsingBlpop_EnsureItTimeout(Jedis jedis) throws ExecutionExceptio
...
         Future<?> future = blockingThread.submit(() -> {
             List<String> result = blockedClient.blpop(1, list1key, list2key, list3key);
-            assertNull(result);
+            assertThat(result).isNull();
         });
         //Check the list is not modified
         jedis.getClient().setSoTimeout(2000);
         List<String> results = jedis.lrange(list2key, 0, -1);
-        assertEquals(0, results.size());
+        assertThat(results).hasSize(0);
```
Comment (inponomarev) [inline src/test/java/com/github/fppt/jedismock/comparisontests/lists/BlockingOperationsTest.java:166]: isEmpty?

File: src/test/java/com/github/fppt/jedismock/comparisontests/sets/SetOperationsTest.java [hunk 3]
```diff
@@ -99,73 +99,69 @@ public void whenPoppingFromASet_EnsureTheSetIsUpdated(Jedis jedis) {
...
         String key = "key-pop";
         jedis.sadd(key, "a");
-        assertTrue(jedis.exists(key));
-        assertEquals(Collections.singleton("a"), jedis.spop(key, 1));
-        assertFalse(jedis.exists(key));
-        assertEquals(0, jedis.spop(key, 0).size());
+        assertThat(jedis.exists(key)).isTrue();
+        assertThat(jedis.spop(key, 1)).containsExactly("a");
+        assertThat(jedis.exists(key)).isFalse();
+        assertThat(jedis.spop(key, 0)).hasSize(0);
```
Comment (inponomarev) [inline src/test/java/com/github/fppt/jedismock/comparisontests/sets/SetOperationsTest.java:113]: isEmpty?

File: src/test/java/com/github/fppt/jedismock/comparisontests/transactions/TestWatchUnwatch.java [hunk 4]
```diff
@@ -87,9 +86,9 @@ public void testWatchWithNoKeyAffection(Jedis jedis) throws ExecutionException,
         runAsync(() -> anotherJedis.set(ANOTHER_KEY, ANOTHER_VALUE)).get();
         transaction.set(FIRST_KEY, SECOND_VALUE);
         List<Object> result = transaction.exec();
-        assertEquals(1, result.size());
-        assertEquals("OK", result.get(0));
-        assertEquals(SECOND_VALUE, jedis.get(FIRST_KEY));
+        assertThat(result).hasSize(1)
```
Comment (inponomarev) [inline src/test/java/com/github/fppt/jedismock/comparisontests/transactions/TestWatchUnwatch.java:89]: `containsExactly` is sufficient here, no need to check for size

File: src/test/java/com/github/fppt/jedismock/comparisontests/transactions/TestWatchUnwatch.java [hunk 5]
```diff
@@ -134,9 +133,9 @@ public void testWatchWithUnwatch(Jedis jedis) throws ExecutionException, Interru
         Transaction transaction = jedis.multi();
         transaction.set(FIRST_KEY, SECOND_VALUE);
         List<Object> result = transaction.exec();
-        assertEquals(1, result.size());
-        assertEquals("OK", result.get(0));
-        assertEquals(SECOND_VALUE, jedis.get(FIRST_KEY));
+        assertThat(result).hasSize(1)
```
Comment (inponomarev) [inline src/test/java/com/github/fppt/jedismock/comparisontests/transactions/TestWatchUnwatch.java:136]: `containsExactly` is sufficient here, no need to check for size

File: src/test/java/com/github/fppt/jedismock/comparisontests/transactions/TestWatchUnwatch.java [hunk 6]
```diff
@@ -156,8 +155,8 @@ public void testWatchWithNoKeyInStorage(Jedis jedis) {
         Transaction transaction = jedis.multi();
         transaction.set(FIRST_KEY, SECOND_VALUE);
         List<Object> result = transaction.exec();
-        assertEquals(1, result.size());
-        assertEquals("OK", result.get(0));
+        assertThat(result).hasSize(1)
```
Comment (inponomarev) [inline src/test/java/com/github/fppt/jedismock/comparisontests/transactions/TestWatchUnwatch.java:158]: `containsExactly` is sufficient here, no need to check for size

File: src/test/java/com/github/fppt/jedismock/TestRedisServer.java [hunk 7]
```diff
@@ -34,7 +36,7 @@ public void testBindErrorPort() throws IOException {
         RedisServer server = RedisServer.newRedisServer(100000);
         try {
```
Comment (inponomarev) [inline src/test/java/com/github/fppt/jedismock/TestRedisServer.java:37]: use check for exception here and in all the similar places

File: src/test/java/com/github/fppt/jedismock/comparisontests/sortedsets/TestZRem.java [hunk 8]
```diff
@@ -31,34 +30,33 @@ public void zRemRemovesKey(Jedis jedis) {
...
         result = jedis.zrem(ZSET_KEY, "myvalue1");
 
-        assertEquals(1L, result);
+        assertThat(result).isEqualTo(1L);
 
         results = jedis.zrange(ZSET_KEY, 0, -1);
 
-        assertEquals(1, results.size());
-        assertEquals("myvalue2", results.get(0));
+        assertThat(results).hasSize(1)
```
Comment (inponomarev) [inline src/test/java/com/github/fppt/jedismock/comparisontests/sortedsets/TestZRem.java:46]: redundand `hasSize(1)`

**Review by inponomarev — CHANGES_REQUESTED** [review]

File: src/test/java/com/github/fppt/jedismock/comparisontests/sets/SetOperationsTest.java [hunk 9]
```diff
@@ -90,7 +93,7 @@ public void whenPoppingFromASet_EnsureTheSetIsUpdated(Jedis jedis) {
         do {
             poppedValue = jedis.spop(key);
             if (poppedValue != null) {
-                assertTrue(mySet.contains(poppedValue), "Popped value not in set");
+                assertThat(mySet).as("Popped value not in set").contains(poppedValue);
```
[the remark this answers is not in the document]
Comment (Alex286756) [reply src/test/java/com/github/fppt/jedismock/comparisontests/sets/SetOperationsTest.java:93]: Single one? Fixed

File: src/test/java/com/github/fppt/jedismock/TestCommandParser.java (hunk 1 above)
Comment (Alex286756) [reply src/test/java/com/github/fppt/jedismock/TestCommandParser.java:85]: Fixed

File: src/test/java/com/github/fppt/jedismock/comparisontests/lists/BlockingOperationsTest.java (hunk 2 above)
Comment (Alex286756) [reply src/test/java/com/github/fppt/jedismock/comparisontests/lists/BlockingOperationsTest.java:166]: Fixed

File: src/test/java/com/github/fppt/jedismock/comparisontests/transactions/TestWatchUnwatch.java (hunk 4 above)
Comment (Alex286756) [reply src/test/java/com/github/fppt/jedismock/comparisontests/transactions/TestWatchUnwatch.java:89]: Fixed

File: src/test/java/com/github/fppt/jedismock/comparisontests/transactions/TestWatchUnwatch.java (hunk 6 above)
Comment (Alex286756) [reply src/test/java/com/github/fppt/jedismock/comparisontests/transactions/TestWatchUnwatch.java:158]: Fixed

File: src/test/java/com/github/fppt/jedismock/TestRedisServer.java (hunk 7 above)
Comment (Alex286756) [reply src/test/java/com/github/fppt/jedismock/TestRedisServer.java:37]: OK

File: src/test/java/com/github/fppt/jedismock/comparisontests/sortedsets/TestZRem.java (hunk 8 above)
Comment (Alex286756) [reply src/test/java/com/github/fppt/jedismock/comparisontests/sortedsets/TestZRem.java:46]: Fixed

**Review by inponomarev — APPROVED** [review]

**Pull request merged**